base/array.jl

```diff
@@ -1251,10 +1251,11 @@ julia> findnext(A,3)
 function findnext(A, start::Integer)
     for i = start:length(A)
         if A[i] != 0
-            return i
```

Comment:
Not related to the feature itself, but I find explicit returns easier to reason about.

Reply:
A nice example is [https://github.com/JuliaLang/julia/blob/644c53cae71d95b6bda3d0db5703a697c58ba096/base/set.jl#L159](`intersect`), which can be improved into:
```
function intersect(s::Set, sets...)
    i = similar(s)
    for x in s
        for t in sets
            !in(x, t) && break false
        else true
        end && push!(i, x)
    end
    return i
end
```

Reply:
Great example @rfourquet ! It could also be
```julia
for x in s
    for t in sets
        !in(x, t) && break
    else
        push!(i, x)
    end
end
```
which doesn't use break-with-value, but does use the `else` feature.

I personally like options with `all` or with comprehensions best, but that's a matter of taste and may not give the same performance (yet).